### PR TITLE
fix various translations

### PR DESCRIPTION
**Pull request by goapunk**
warning: contains migrations

### CODE REVIEW

File: apps/measures/templates/apps_measures/includes/measures_list_item.html [hunk 1]
```diff
@@ -25,8 +25,8 @@ <h3 class="title">{{ content.page_title|truncatechars:102 }}</h3>
             </p>
 
             <a href="{% pageurl content %}" class="more" data-mainlink="true" title="This is a single Teaser">
-                {% trans 'More' %}
-                <span class="aural">{% trans 'to: ' %} {{ content.translated_title }}</span>
```
Comment (goapunk) [inline apps/measures/templates/apps_measures/includes/measures_list_item.html:29]: translated_title always seemed to be empty? not sure if that was the correct field so changed it to page_title

Comment (goapunk) [reply apps/measures/templates/apps_measures/includes/measures_list_item.html:29]: and for me the SR reads "more to <page title>", somehow "more about <page title>" sounds better to me. What do you think?

**Review by Rineee — APPROVED** [review]
Lgtm! Although not sure about the screenreader thing, maybe @philli-m could have a quick look at that?

File: apps/measures/templates/apps_measures/includes/measures_list_item.html (hunk 1 above)
Comment (philli-m) [reply apps/measures/templates/apps_measures/includes/measures_list_item.html:29]: It's just what the styleguide does to more links but happy with the change

**Review by philli-m — APPROVED** [review]
cool

**Pull request merged**